docker/humble_gzharmonic/dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM aerostack2/nightly-humble:latest
+
+# Install common tools
+RUN apt update \
+    && apt-get update && apt-get install -y \
+    lsb-release wget gnupg software-properties-common \
+    && rm -rf /var/lib/apt/lists/* 
+
+# Gazebo Fortress clean uninstall
+RUN apt remove ignition* -y \
+    && apt autoremove -y
+
+# Install Gazebo Harmonic
```

Comment:
Use more RUN instructions instead of concatenating commands with &&. This is helpful for debugging when a build fails.